content/riak/kv/2.2.6/developing/data-types/gsets.md

```diff
@@ -450,19 +440,19 @@ var_dump($gset->getData());
 ```
 
 ```python
-cities_set.dirty_value
+gset.dirty_value
 
 # The value fetched from Riak is always immutable, whereas the "dirty
 # value" takes into account local modifications that have not been
 # sent to the server. For example, where the call above would return
 # frozenset(['Toronto', 'Hamilton', 'Ottawa']), the call below would
 # return frozenset([]).
 
-cities_set.value
+gset.value
 
 # To fetch the value stored on the server, use the call below. Note
 # that this will clear any unsent additions or deletions.
```

Comment:
Deletions are not possible in gsets

Reply:
Adjusted to remove reference to deletions